src/views/IndexView.js

```diff
@@ -75,8 +75,6 @@ class IndexView extends PureComponent {
           <iframe
             title="star"
             src="https://ghbtns.com/github-btn.html?user=recharts&repo=recharts&type=star&count=true&size=median"
-            frameBorder="0"
-            scrolling="0"
```

Comment:
How is this change motivated? It seems unrelated.

Reply:
they are deprecated attributes https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#deprecated_attributes

Reply:
added some CSS to replace these deprecated attrs